LivingCity/benchmarking/runBenchmarks.py

```diff
@@ -182,20 +183,26 @@ def benchmark_one_run(number_of_run, benchmark_name, network_path):
 
     # ================ Metadata ================
     log("Saving metadata...")
-    metadata = {'elapsed_time_LivingCity': elapsed_time_LivingCity}
+    metadata = {'elapsed_time_simulation': elapsed_time_simulation}
     for output_line in full_output[2:].split("\\n"):
         if re.search('Number of blocks: '.format(component_name), output_line, flags=0):
             metadata['number_of_blocks'] = int(output_line.split()[-1])
         if re.search('Number of threads per block: '.format(component_name), output_line, flags=0):
             metadata['number_of_threads_per_block'] = int(output_line.split()[-1])
     
+    df_nodes = pd.read_csv(os.path.join(network_path, "nodes.csv"))
+    metadata['number_of_nodes'] = len(df_nodes)
+
+    df_edges = pd.read_csv(os.path.join(network_path, "edges.csv"))
+    metadata['number_of_edges'] = len(df_edges)
+
     with open("benchmarking/metadata_{}.json".format(benchmark_name),"w") as metadata_file:
         json.dump(metadata, metadata_file)
 
     log("Done")
 
 
-def benchmark_multiple_runs(number_of_runs, benchmark_name = "benchmarks", network_path = "berkeley_2018/new_full_network/"):
+def benchmark_multiple_runs_and_save_csv(number_of_runs, benchmark_name = "benchmarks", network_path = "berkeley_2018/new_full_network/"):
```

Comment:
In the future, that default value might be a configuration parameter and / or an environment variable.